deps/chakrashim/include/v8-debug.h

```diff
@@ -46,25 +81,40 @@ class V8_EXPORT Debug {
   typedef void (*DebugMessageDispatchHandler)();
   typedef void (*MessageHandler)(const Message& message);
 
+  static MessageQueue messageQueue;
+  static MessageHandler handler;
+  static void EnableDebug();
   static void DebugBreak(Isolate *isolate = NULL) {}
   static void SetDebugMessageDispatchHandler(
     DebugMessageDispatchHandler handler, bool provide_locker = false) {}
-  static bool EnableAgent(
-    const char *name = NULL, int port = 0, bool wait_for_connection = false);
-  static void Dispose();
-  static void DisableAgent() {}
-  static bool IsAgentEnabled();
+  static void Dispose() {}
   static void ProcessDebugMessages(Isolate* isolate) {}
   static Local<Context> GetDebugContext(Isolate* isolate);
-  static void SetMessageHandler(Isolate* isolate, MessageHandler handler) {}
+  static void SetMessageHandler(Isolate* isolate, MessageHandler handler);
   static void SendCommand(Isolate* isolate,
                           const uint16_t* command, int length,
-                          ClientData* client_data = NULL) {
-  }
+                          ClientData* client_data = NULL);
   static MaybeLocal<Value> GetMirror(Local<Context> context,
                                      Handle<Value> obj) {
     return MaybeLocal<Value>();
   }
+  static void StartDebugging(JsRuntimeHandle runtime);
```

Comment:
I recall v8::Debug is a public class in v8 headers. These look like deviate from v8 signature. Do we need to put them here, or can we move them to private headers?

Same question to other new classes in this file.